openwisp_utils/admin_theme/templatetags/ow_tags.py

```diff
@@ -15,3 +16,9 @@ def ow_create_filter(cl, spec, total_filters):
             'show_button': total_filters > 4,
         }
     )
+
+
+@register.filter
+@stringfilter
+def join_string(value):
+    return value.lower().replace(' ', '-')
```

Comment:
can you please add a docstring to this filter to explain how it is used and why we need it? It's not super clear to me.